beacon-chain/state/stateutil/blocks.go

```diff
@@ -32,6 +33,88 @@ func BlockHeaderRoot(header *ethpb.BeaconBlockHeader) ([32]byte, error) {
 	return bitwiseMerkleize(fieldRoots, uint64(len(fieldRoots)), uint64(len(fieldRoots)))
 }
 
+// BlockRoot returns the block hash tree root of the provided block.
+func BlockRoot(blk *ethpb.BeaconBlock) ([32]byte, error) {
```

Comment:
I'm a bit torn on the naming. `BlockRoot` sounds a pure getter and its retrieving the the root within the block with no real work. Should we be consistent with `HashTreeRootState` and name it `HashTreeRootBlock` ?  This applies to attestations as well

Reply:
I was trying to put int more in line to how naming was done in this package, all the function names are basically `data_structure`Root. Maybe we need to standardize naming here, although that will have to be in another pr

Reply:
Are you saying `HashTreeRootState` should be named `StateRoot`? 

I'm ok with that, as long as the 2 are consistent